plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -465,19 +533,19 @@ enterComputeCek
     -> Context uni fun
     -> CekValEnv uni fun
     -> Term Name uni fun ()
-    -> CekM s (Term Name uni fun ())
+    -> CekM uni fun s (Term Name uni fun ())
 enterComputeCek costs = computeCek where
     -- | The computing part of the CEK machine.
     -- Either
     -- 1. adds a frame to the context and calls 'computeCek' ('Force', 'Apply')
     -- 2. calls 'returnCek' on values ('Delay', 'LamAbs', 'Constant', 'Builtin')
-    -- 3. returns 'EvaluationFailure' ('Error')
+    -- 3. throws 'EvaluationFailure' ('Error')
     -- 4. looks up a variable in the environment and calls 'returnCek' ('Var')
     computeCek
```

Comment:
There's a warning near the top of the file about not exporting `computeCek`, but that's now not possible with `computeCek` being defined inside `enterCek`.  I guess the comment's still worth having though in case someone decides to reorganise things in a few years.